--- a/arch/arm/dts/suniv.dtsi	2019-07-31 02:16:16.000000000 +0000
+++ b/arch/arm/dts/suniv.dtsi	2023-11-02 02:33:03.936035808 +0000
@@ -126,6 +126,11 @@
 				function = "uart0";
 			};
 
+                        uart1_pa_pins: uart1-pa-pins {
+                                pins = "PA2", "PA3";
+                                function = "uart1";
+                        };
+
 			mmc0_pins: mmc0-pins {
 				pins = "PF0", "PF1", "PF2", "PF3", "PF4", "PF5";
 				function = "mmc0";
